pom.xml

```diff
@@ -16,47 +16,47 @@
         <sonar.coverage.jacoco.xmlReportPaths>../multiapps-controller-coverage/target/site/jacoco-aggregate/jacoco.xml</sonar.coverage.jacoco.xmlReportPaths>
         <junit.version>5.10.2</junit.version>
         <junit-platform.version>1.10.2</junit-platform.version>
-        <mockito.version>4.11.0</mockito.version>
-        <derby.version>10.14.2.0</derby.version>
+        <mockito.version>5.11.0</mockito.version>
+        <derby.version>10.16.1.1</derby.version>
         <h2.version>2.2.224</h2.version>
         <servlet-api.version>4.0.4</servlet-api.version>
         <log4j.version>2.23.1</log4j.version>
         <slf4j.version>2.0.12</slf4j.version>
-        <commons-io.version>2.15.1</commons-io.version>
+        <commons-io.version>2.16.1</commons-io.version>
         <commons-fileupload.version>1.5</commons-fileupload.version>
         <commons-collections4.version>4.4</commons-collections4.version>
         <snakeyaml.version>2.2</snakeyaml.version>
         <httpclient.version>4.5.14</httpclient.version>
-        <httpcore.version>4.4.13</httpcore.version>
+        <httpcore.version>4.4.16</httpcore.version>
         <eclipselink.version>2.7.14</eclipselink.version>
         <flowable.version>6.8.0</flowable.version>
-        <mybatis.version>3.5.7</mybatis.version>
+        <mybatis.version>3.5.16</mybatis.version>
         <spring.version>5.3.33</spring.version>
         <spring-security.version>5.8.11</spring-security.version>
-        <json-smart.version>2.4.9</json-smart.version>
+        <json-smart.version>2.5.1</json-smart.version>
         <hikari.version>5.1.0</hikari.version>
         <jackson.version>2.17.0</jackson.version>
         <jackson.databind.version>2.17.0</jackson.databind.version>
-        <liquibase.version>4.25.1</liquibase.version>
+        <liquibase.version>4.27.0</liquibase.version>
         <liquibase-slf4j.version>5.0.0</liquibase-slf4j.version>
         <cloudfoundry-client.version>2.41.0</cloudfoundry-client.version>
-        <swagger.version>1.6.9</swagger.version>
+        <swagger.version>1.6.14</swagger.version>
         <jclouds.version>2.6.0</jclouds.version>
-        <guava.version>32.0.1-jre</guava.version>
-        <resilience4j.version>1.7.1</resilience4j.version>
+        <guava.version>33.1.0-jre</guava.version>
```

Comment:
Guava is used by jclouds and needs to check does everything works on all IaaS providers again.